.github/workflows/publish-ci.yml

```diff
@@ -98,3 +105,9 @@ jobs:
           user: __token__
           password: ${{ secrets.PYDEEPHAVEN_PYPI_TOKEN }}
           packages_dir: py/client/build/wheel/
+
+      - name: Publish @deephaven/jsapi-types to npmjs
+        if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.DEEPHAVENBOT_NPM_TOKEN }}
+        run: npm publish --tag latest web/client-api/types/build/deephaven-jsapi-types-*.tgz
```

Comment:
Yes - this is the way to say "here's a finished tarball, go ahead and deploy it". The work of making the tarball is done inside of docker (so that we guarantee it builds the way we expect, that it will do on our own machines), so this npm invocation just has access to the finished tarball and the secret.